### ISSUE TITLE
Keyboard trap in Editing Interface

### ISSUE THREAD

**Issue by mgifford** [gctools-outilsgc/gcpedia#138]
It isn't clear how to navigate the default **Edit** screen as a keyboard only user.  Tabs don't work, arrows have a different functionality than expected, escape allows you to escape (which is good) but not save. 

I think it would work better if you **Edit Source**, but it isn't clear as a user why you would use one over the other. 

<img width="906" alt="Screenshot of Edit page" src="https://user-images.githubusercontent.com/116832/72950397-422f4800-3d59-11ea-8131-49fe7d1d883c.png">

**Comment by nschonni** [comment]
Is this an issue with implementation or MediaWiki? Probably good to indicate which deployment too, since I'm not sure GCpedia is running the same version as GCcollab

**Comment by mgifford** [comment]
I've been doing my testing on GCWiki https://wiki.gccollab.ca because it is public & easy.

I had assumd that GCPedia & GCWiki would be running parallel versions of MediaWiki. That might be a bad assumption. 

This might be a general problem with MediaWiki, but if this is the case, it should be reported upstream.  Accessibility issues that are known and of concern in GCPedia should be opened to link to upstream issues that you are working on fixing upstream. 

This is probably a known issue in MediaWiki, but maybe not. There is still the question about how the GCTools team deals with accessibility bugs in tools like GCPedia, which is essentially critical infrastructure for internal communications.
